### PR TITLE
Enhance SSH service

### PR DESCRIPTION
**Pull request by nebilbenmabrouk**

### CODE REVIEW

File: SSH/resources/catalog/SSH_Service.xml [hunk 1]
```diff
@@ -81,12 +98,17 @@ $INSTANCE_NAME (Required): service instance name ]]>
...
-    <task fork="true" name="wait_for_stop">
+    <task name="wait_for_stop" 
+    
+    
+    
+    
+    fork="true">
       <description>
-        <![CDATA[ A template task that sends a ready notification for all the signals specified in the variable SIGNALS, then loops until one signal among those specified is received by the job. ]]>
+        <![CDATA[ A task that declares being ready for receiving the signal "Terminate_Job_and_Service", then iterates to check whether this signal is received (for example when sent by user from the Workflow Execution portal). When this signal is received, the task terminates. ]]>
```
Comment (mboussaa) [inline SSH/resources/catalog/SSH_Service.xml:108]: Terminate_Job_and_Service => Terminate_Job_And_Services

Comment (mboussaa) [reply SSH/resources/catalog/SSH_Service.xml:108]: then iterates to check whether this signal is received => then loops over to check if this signal is received

File: SSH/resources/catalog/SSH_Service.xml [hunk 2]
```diff
@@ -81,12 +98,17 @@ $INSTANCE_NAME (Required): service instance name ]]>
...
       </description>
       <variables>
-        <variable inherited="false" name="SIGNALS" value="Terminate_job_and_service"/>
+        <variable name="SIGNALS" value="Terminate_Job_and_Service" inherited="false" />
```
Comment (mboussaa) [inline SSH/resources/catalog/SSH_Service.xml:111]: I am not sure if it should be Terminate_Job_And_Services as in WE.

File: SSH/resources/catalog/SSH_Service.xml (hunk 1 above)
Comment (nebilbenmabrouk) [reply SSH/resources/catalog/SSH_Service.xml:108]: - It is a single service
- Both sentences are correct

File: SSH/resources/catalog/SSH_Service.xml (hunk 2 above)
Comment (nebilbenmabrouk) [reply SSH/resources/catalog/SSH_Service.xml:111]: Again it is a single service

File: SSH/resources/catalog/SSH_Service.xml [hunk 3]
```diff
@@ -1,38 +1,47 @@
...
+    <variable name="INSTANCE_NAME" value="ssh-terminal-${PA_JOB_ID}" />
+    <variable name="TARGET_HOST" value="" model="PA:MODEL_FROM_URL(${PA_SCHEDULER_REST_PUBLIC_URL}/rm/model/hosts)"/>
+    <variable name="PROXYFIED" value="false" model="PA:Boolean"/>
+    <variable name="SSH_PORT" value="22" model="PA:NOT_EMPTY_STRING"/>
   </variables>
   <description>
-    <![CDATA[ Deploy a SSH terminal on web browser using the Wetty tool.
-The service can be started using the variable:
-$INSTANCE_NAME (Required): service instance name ]]>
+    <![CDATA[ Deploy a SSH terminal on web browser using the Wetty tool. This service takes as input:
```
Comment (mboussaa) [inline SSH/resources/catalog/SSH_Service.xml:12]: Deploy a SSH => Deploy an SSH 
But not sure, I found it easier with "an"

**Pull request merged**